### ISSUE TITLE
Recommend putting multiple COPY lines on the same line to reduce layers

### ISSUE THREAD

**Issue by nitrocode** [hadolint/hadolint#364]
- [] This is a bug report
- [x] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior

```bash
$ hadolint Dockerfile
Try to move multiple sequential COPY commands to the same line
```

### Actual behavior

Doesn't flag it

### Steps to reproduce the behavior

See above

```bash
Haskell Dockerfile Linter 1.17.2-no-git
```

**Dockerfile** (if relevant)

```Dockerfile
COPY requirements.txt
COPY start.sh
```

**Additional environment details (OS, stack version, etc.)**

osx
